.github/renovate.json

```diff
@@ -0,0 +1,22 @@
+{
+    "extends": [
+        "config:base"
+    ],
+    "labels": ["dependencies"],
+    "packageRules": [{
+            "matchManagers": ["github-actions"],
+            "addLabels": ["github_actions"]
+        },
+        {
+            "matchManagers": ["git-submodules"],
+            "addLabels": ["git-submodules"],
+            "automerge": true,
+            "automergeType": "branch"
+        }
+    ],
+    "git-submodules": {
+        "enabled": true
+    },
+    "timezone": "Etc/UTC",
+    "schedule": ["before 2am"]
+}
```

Comment:
Missing a new line

Reply:
VS Code autmatically removes newlines at the end of json files.

Is it really required?

Reply:
We generally have those for all files enabled via the editorconfigs. I think this repository does too.

Reply:
You are correct. The weird thing is I have the editorconfig plugin installed on vscode but it seem to either be ignored or overridden by another setting.

Nonetheless I fixed it manually.